x/lockup/keeper/admin_keeper.go

```diff
@@ -0,0 +1,32 @@
+package keeper
+
+import (
+	"github.com/c-osmosis/osmosis/x/lockup/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// RageQuit unlock previous lockID and create a new lock with newCoins with same duration and endtime
+func (ak AdminKeeper) RageQuit(ctx sdk.Context, lockID uint64, newCoins sdk.Coins) error {
```

Comment:
Let's call it something different.  RageQuit is name specifically for governance purpose.  Maybe we call this Relock?

Also, since this is the AdminKeeper, it should probably just have the ability to plain `BreakLock`, without necessarily needing to create a new one as well.

Reply:
Updated to `Relock` and added `BreakLock`